wazo_acceptance/steps/ha.py

```diff
@@ -46,3 +74,18 @@ def when_i_enable_the_ha_as_slave_on_instance(context, instance):
         'node_type': 'slave',
         'remote_address': context.wazo_config['master_host']
     })
+
+
+@when('I start the replication from "{instance_master}" to "{instance_slave}"')
+def when_i_start_the_replication_from_instancemaster_to_instanceslave(context, instance_master, instance_slave):
+    master_context = getattr(context.instances, instance_master)
+    slave_host = master_context.wazo_config['slave_host']
+    master_context.remote_sysutils.send_command(['xivo-master-slave-db-replication', slave_host], check=True)
+    slave_context = getattr(context.instances, instance_slave)
+    auth.renew_auth_token(slave_context)
+
+
+@then('there is a user "{username}" on "{instance}"')
+def then_there_is_a_user_username_on_instance(context, username, instance):
+    context = getattr(context.instances, instance)
+    assert context.helpers.user.get_by(username=username)
```

Comment:
What is the advantage to add `asset`?

Reply:
I find it clearer: it says "that's what this step expects". An alternative would be a `try: except ...: fail()` around the `get_by`, but it would be harder to read.